packages/web3torrent/src/clients/payment-channel-client.ts

```diff
@@ -452,7 +452,7 @@ export class PaymentChannelClient {
       );
     } catch (e) {
       if (e.message === 'User declined') {
-        log.info('User declined budget creation');
+        log.trace('User declined budget creation');
```

Comment:
I think it's worth taking some time to think about what should be `info`, `trace`, etc. 

For instance, this is catching an error, and it doesn't happen repeatedly. So I don't think it should be `trace`.

A good rule of thumb seems to be:
A. things that happen repeatedly should tend have log level equal to `debug` or `trace`
B. things we might want to know about, to see what's going on, should have log level equal to `info`, `warn` or `error`

Did you apply some kind of rule of thumb like this?

Reply:
While this is catching an "error" technically, it's essentially logging out that the user has declined the budget creation by clicking a button.

In my mind this is useful information to have if we're trying to debug what went wrong in some situation. However there doesn't seem to be a huge benefit to logging this out in the normal process, as we're essentially echoing the action the user took in the UI.

My rough rule of thumb is that `info` is used for messages that might be of interest for advanced users and provide insight into what is going on. `trace` should be used for messages that help us trace what was going on when something went wrong. (Although I only did a quick pass through `web3torrent` so the log level might not be right for all of them.